### DISCUSSION TITLE
M1 / Apple Silicon - Big Sur 11.6.7 - Scripting doesn't work

### DISCUSSION THREAD

**Discussion by mrr11k** [asmvik/yabai#1323] [Q&A] [answered]
Has anyone of you got it to run Big Sur on Apple Silicon? 
The [wiki](https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection) for disabling SIP, doesn't state any guide for this config. Just for 12.x. 

The general space partition works, but I can't run any command and so can't setup keybinds with skhd.

Any help appreciated. :)

**Comment by asmvik** [comment, marked as answer]
The scripting-addition (features listed [here](https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection)) does not work on Big Sur on Apple Silicon.
